CONTRIBUTING.md

```diff
@@ -85,6 +85,25 @@ All PRs require an approval from at least one CODEOWNER before merge. PRs which
 - If you sat down with the PR submitter and did a pairing review please note that in the `Approval`, or your PR comments.
 - If you are only making "surface level" reviews, submit any notes as `Comments` without adding a review.
 
+### Commit Messages
+
+Commit messages should be [conventional](https://www.conventionalcommits.org/en/v1.0.0/).
```

Comment:
Do all the commits in the PR need to follow this convention or the one that matters is the commit message that is entered when squashing the PR? Because in that case, we can have more relaxed rules for the commits while you're working on the PR, and then be careful when squashing the PR, which is something that we have control of.

Reply:
yes you're right it's only the final squashed commit that matters.